test/Microsoft.AspNetCore.Mvc.FunctionalTests/RoutingTestsBase.cs

```diff
@@ -1475,5 +1474,10 @@ protected static LinkBuilder LinkFrom(string url)
         {
             return new LinkBuilder(url);
         }
+
+        protected virtual void ConfirmNotFoundOrNotAllowed(HttpResponseMessage response)
```

Comment:
nit: `Assert...` instead of `Confirm...`

Reply:
I think the name could be more descriptive: AssertCorsRejectionStatusCode